src/network/netdev/netdev-gperf.gperf

```diff
@@ -230,3 +231,12 @@ WireGuardPeer.PresharedKeyFile,           config_parse_wireguard_preshared_key_f
 WireGuardPeer.PersistentKeepalive,        config_parse_wireguard_keepalive,          0,                             0
 Xfrm.InterfaceId,                         config_parse_uint32,                       0,                             offsetof(Xfrm, if_id)
 Xfrm.Independent,                         config_parse_bool,                         0,                             offsetof(Xfrm, independent)
+Batadv.Aggregation,                       config_parse_bool,                         0,                             offsetof(Batadv, aggregation)
+Batadv.BridgeLoopAvoidance,               config_parse_bool,                         0,                             offsetof(Batadv, bridge_loop_avoidance)
+Batadv.DistributedArpTable,               config_parse_bool,                         0,                             offsetof(Batadv, distributed_arp_table)
+Batadv.Fragmentation,                     config_parse_bool,                         0,                             offsetof(Batadv, fragmentation)
+Batadv.GatewayMode,                       config_parse_batadv_gateway_mode,          0,                             offsetof(Batadv, gateway_mode)
+Batadv.GatewayBandwithDown,               config_parse_badadv_bandwidth,             0,                             offsetof(Batadv, gateway_bandwidth_down)
+Batadv.GatewayBandwithUp,                 config_parse_badadv_bandwidth,             0,                             offsetof(Batadv, gateway_bandwidth_up)
+Batadv.HopPenalty,                        config_parse_uint8,                        0,                             offsetof(Batadv, hop_penalty)
+Batadv.OrigInterval,                      config_parse_uint32,                       0,                             offsetof(Batadv, orig_interval)
```

Comment:
please don't abbrev needlessly. There's no shame in typing out OriginalInterval=

Reply:
I just adopted the slang of `batctl` here so people can easier find the options. But will rename to OriginatorInterval= :).